pyproject.toml

```diff
@@ -86,6 +86,7 @@ werkzeug = "2.3.6"
 wtforms = "2.2.1"
 xlsxwriter = "1.2.2"
 tzlocal = "4.3.1"
+e6data-python-connector = "1.1.9"
```

Comment:
You should add this to [tool.poetry.group.all_ds.dependencies]

Reply:
@konnectr Nice catch! I missed it 😅

Reply:
> You should add this to [tool.poetry.group.all_ds.dependencies] and remove from here

Alright, will do that. Thanks